ta4j/src/main/java/org/ta4j/core/indicators/pivotpoints/FibonacciReversalIndicator.java

```diff
@@ -28,24 +28,52 @@
 import java.util.List;
 
 /**
- * Fibonacci Pivot Support 3 Indicator.
+ * Fibonacci Reversal Indicator.
  * <p>
  * @author team172011(Simon-Justus Wimmer), 09.10.2017
  * @see <a href="http://stockcharts.com/school/doku.php?id=chart_school:technical_indicators:pivot_points">chart_school: pivotpoints</a>
  */
-public class FibonacciPivotSupport3Indicator extends RecursiveCachedIndicator<Decimal> {
+public class FibonacciReversalIndicator extends RecursiveCachedIndicator<Decimal> {
 
     private PivotPointIndicator pivotPointIndicator;
-    private static final Decimal FIB_FACOTR = Decimal.ONE;
+    private PivotLevel level;
+
+    public enum PivotLevel{
+        RESISTANCE_3(Decimal.ONE, true),
+        RESISTANCE_2(Decimal.valueOf(0.618), true),
+        RESISTANCE_1(Decimal.valueOf(0.382), true),
+        SUPPORT_1(Decimal.valueOf(0.382), false),
+        SUPPORT_2(Decimal.valueOf(0.618), false),
+        SUPPORT_3(Decimal.ONE, false);
+
+        private Decimal fibFactor;
+        private boolean isResistance;
```

Comment:
No need for "isResistance". Fibonacci Numbers are equal for both resistance and support. Please look at my solution described at https://github.com/ta4j/ta4j/pull/49.

Reply:
```
public enum FibonacciFactor {
		Factor1(Decimal.valueOf(0.382)), 
		Factor2(Decimal.valueOf(0.618)), 
		Factor3(Decimal.ONE);

		FibonacciFactor(Decimal factor) {
			this.factor = factor;
		}
		private final Decimal factor;
	}
```

Reply:
ah, thought it is always the same -.-